Test/comp/AutoInit.dfy

```diff
@@ -286,44 +285,7 @@ module ImportedTypes {
   }
 }
 
-module GhostWitness {
```

Comment:
Why is this test removed? Which part fails?

Reply:
The entire test is erroneous because once you use the appropriate axioms and checks, you cannot define partial functions without read effects in this way.